e2e/cypress/support/step_definitions/runner/i_upload_a_file_that_string.js

```diff
@@ -0,0 +1,11 @@
+import { When } from "@badeball/cypress-cucumber-preprocessor";
+
+When("I upload a file that {string}", (string) => {
```

Comment:
We should be more explicit in what a "bad" file is if this is going to be a common step definition. "I upload a file that {string} the OCR check".


Alternatively naming "bad" or "passes" is fine in the context of testing OCR, but you would put all the steps in `e2e/cypress/e2e/runner/imageQualityPlayback.js`.